cmd/newallocation.go

```diff
@@ -237,6 +237,22 @@ var newallocationCmd = &cobra.Command{
 			fileOptionParams.ForbidRename.Value = forbidRename
 		}
 
+		var storageVersion int64
+		if flags.Changed("storage_version") {
+			storageVersion, err = flags.GetInt64("storage_version")
+			if err != nil {
+				log.Fatal("invalid forbid_upload: ", err)
+			}
+		}
+
+		var managingWallet string
```

Comment:
Why did you add managingWallet in newAllocationRequest?

Reply:
Removed